web/src/lib/utils/cast/gcast-destination.svelte.ts

```diff
@@ -0,0 +1,264 @@
+import { CastDestinationType, CastState, type CastDestination } from '$lib/managers/cast-manager.svelte';
+import 'chromecast-caf-sender';
+import { Duration } from 'luxon';
+
+const FRAMEWORK_LINK = 'https://www.gstatic.com/cv/js/sender/v1/cast_sender.js?loadCastFramework=1';
```

Comment:
Is it possible to bundle this file in our build instead of having the application call out to google whenever the cast button is used?

Reply:
We could but the recommendation from google is that you load it directly. Do you want me to add it?

Reply:
@alextran1502 agreed that we should just keep this as is since the cast framework calls out to google anyway regardless if we host this file ourselves.